mon-pix/app/routes/assessments.js

```diff
@@ -10,7 +10,7 @@ export default class AssessmentsRoute extends Route {
 
   afterModel(model) {
     if (model.isCertification) {
-      model.title = this.intl.t('page-title.certification-number', { certificationNumber: model.title });
+      model.title = this.intl.t('pages.challenge.certification.title', { certificationNumber: model.title });
```

Comment:
pourquoi pas seulement:

```suggestion
      model.title = this.intl.t('pages.certification.title', { certificationNumber: model.title });
```

Reply:
Dans ma vision : 
on reste sur la page Challenge dans tous les cas, qui peut être une épreuve pour une certif ou pour autre chose. Certification, ça peut être la page avec le certificat finale, je ne trouve pas cela claire